assets/css/src/generic/_forms-yesno.scss

```diff
@@ -0,0 +1,80 @@
+.mailpoet-form-yesno {
+  background: $color-input-background;
+  border: 1px solid $color-input-border;
+  border-radius: 4px;
```

Comment:
I guess the value of `border-radius` is shared across multiple components (buttons, inputs, etc.). We could probably make it a variable.

Reply:
👍 updated